src/auth/auth0-plugin.js

```diff
@@ -64,11 +64,12 @@ export const useAuth0 = ({
 
     async created() {
       this.auth0Client = await createAuth0Client({
-        ...pluginOptions,
         domain: pluginOptions.domain,
-        client_id: pluginOptions.clientId,
-        audience: pluginOptions.audience,
-        redirect_uri: redirectUri,
+        clientId: pluginOptions.clientId,
+        authorizationParams: {
+          audience: pluginOptions.audience,
+          redirect_uri: redirectUri,
+        },
```

Comment:
We introduced `authorizationParams`: https://github.com/auth0/auth0-spa-js/blob/master/MIGRATION_GUIDE.md#introduction-of-authorizationparams